test/fixtures/templates_settings.yml

```diff
@@ -0,0 +1,40 @@
+templates_verbose:
```

Comment:
I'm not a huge fan of fixtures - it was me that did the initial work to move to FactoryGirl and we're *still* trying to get rid of the rest :P. So, I don't really want to add *more* fixtures if we can avoid it.

Is there any reason not to use https://github.com/theforeman/foreman/blob/develop/test/factories/settings.rb?